scripts/funcotator/data_sources/createLiftoverForB37ToHg38.sh

```diff
@@ -0,0 +1,567 @@
+#!/usr/bin/env bash
```

Comment:
How was this verified to work?  Did you give the results to the comms team to post somewhere?

Reply:
For now the resulting liftover file hasn't been validated.  The liftover of gnomAD resulting from using this file is going to be QC'd, then it will have been verified.

I haven't yet given them the liftover file.

Reply:
I'm working with them now.